api_cgn.yaml

```diff
@@ -0,0 +1,88 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: Carta Giovani Nazionale API
+host: localhost
+basePath: /api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+paths:
+  "/cgn/status":
+    get:
+      operationId: getCgnStatus
+      summary: |
+        Get the CGN status details 
+        by the provided fiscal code
+      responses:
+        "200":
+          description: CGN status details.
+          schema:
+            $ref: "#/definitions/CgnStatus"
+        "401":
+          description: Wrong or missing function key.
+        "404":
+          description: No CGN found.
+        "500":
+          description: Service unavailable.
+          schema:
+            $ref: "#/definitions/ProblemJson"
+
+definitions:
```

Comment:
aren't these defs exposed by io-fn-cng? can we reference them instead?

Reply:
I've tried it but `io-utils` failed to generate models. I guess it's for a circular reference problem

Reply:
may youg dig more into it and try to fix ? (even in io-utils)

Reply:
Fixed in [https://github.com/pagopa/io-backend/pull/751/commits/c97db8064d23a437f7f028b7dbb5822ca6174e89](https://github.com/pagopa/io-backend/pull/751/commits/c97db8064d23a437f7f028b7dbb5822ca6174e89). There is another [PR](https://github.com/pagopa/io-backend/pull/751)